global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "3.0.100-preview-009812"
+    "dotnet": "3.0.100-preview4-010564"
```

Comment:
My ears perked up for source-build implications of this change, since preview4 isn't out the door.

As long as the final released preview4 will be able to build the repo too, I think this is fine. If not, then I *think* it's *ok* to have some unsourcebuildable commits, but by the time preview5 comes around the final preview4 version should be here.

/cc @crummel @dseefeld

Reply:
+1, as long as this is replaced by the final preview4 version before preview5 it should be good.

Reply:
Filed https://github.com/dotnet/core-sdk/issues/1312 to track this.